gpii/node_modules/flowManager/src/SystemUtils.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * GPII Utilities
+ *
+ * Copyright 2018 OCAD University
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii");
+
+gpii.reservedGpiiKeys = ["noUser", "restore"];
+
+/**
+ * Check if the given GPII key is a system reserved key.
+ * @param {String} gpiiKey - The GPII key.
+ * @return {Boolean} Return true if the given GPII key is a system reserved key. Otherwise, return false.
+ */
+gpii.isReservedGpiiKey = function (gpiiKey) {
+    return gpii.reservedGpiiKeys.indexOf(gpiiKey) !== -1;
```

Comment:
Note that it is now safe to use.... includes!
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes